src/applications/mhv-secure-messaging/containers/EditContactList.jsx

```diff
@@ -175,88 +203,101 @@ const EditContactList = () => {
         Select the teams you want to show in your contact list when you start a
         new message.{' '}
       </p>
-      {showBlockedTriageGroupAlert && (
-        <div
-          className={`${allFacilities?.length > 1 &&
-            'vads-u-margin-bottom--4'}`}
-        >
-          <BlockedTriageGroupAlert
-            blockedTriageGroupList={blockedRecipients}
-            alertStyle={BlockedTriageAlertStyles.ALERT}
-            parentComponent={ParentComponent.CONTACT_LIST}
-          />
+      {!allTriageTeams ? (
+        <div>
+          {' '}
+          <VaAlert
+            role="alert"
+            aria-live="polite"
+            class="vads-u-margin-y--4"
+            status="error"
+            visible
+            data-testid="contact-list-empty-alert"
+          >
+            <h2 className="vads-u-margin-y--0">
+              We can’t load your contact list right now
+            </h2>
+            <p>
+              We’re sorry. There’s a problem with our system. Try again later.
+            </p>
+            <p>
+              If it still doesn’t work, call us at{' '}
+              <VaTelephone contact={CONTACTS.MY_HEALTHEVET} /> (
+              <VaTelephone contact={CONTACTS['711']} tty />
+              ). We’re here Monday through Friday, 8:00 a.m. to 8:00 p.m. ET.
+            </p>
+          </VaAlert>
+          {goBackButton()}

```

Comment:
If changed from a function to a component, render here like so 👇🏻 
```suggestion
          <GoBackButton />

```